.gitignore

```diff
@@ -1,24 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-*.pyc
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
 
-hg-data/histogram_tools.py
+# testing
+/coverage
 
-hg-data/Histograms.json
+# production
+/build
 
-hg-data/nsDeprecatedOperationList.h
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
 
-hg-data/UseCounters.conf
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Maybe also *package-lock.json* if you are planning to use Yarn.

Reply:
So that other contributors don't mistakenly commit it.

Reply:
Ah, it looks like you committed *package-lock.json*. So maybe ignore *yarn.lock*.